README.md

```diff
@@ -1,3 +1,11 @@
 # Javascript Basics
 
 This repository was made for beginners to start learning Javascript from Scratch
+
```

Comment:
@MarineP31  Does the linting and fixing problems done by one command only

Reply:
yes you just have to use the command `npm run lint` in your terminal to see all the eslint issues but I can add `--fix` to the command line to fix automatically the problem

Reply:
@MarineP31  Go Ahead and add the command for fix also